### DISCUSSION TITLE
how to install programs?

### DISCUSSION THREAD

**Discussion by Ilyha4ws** [Nitrux/nitrux-bug-tracker#160] [Q&A] [answered]
Good afternoon, can you tell me how to install google chrome?and how to install programs in the usual way by downloading deb, or via PPA?

**Comment by UriHerrera** [comment, marked as answer]
Please read the documentation on the website. The [Notes](https://nxos.org/notes/notes-nitrux-3-3-0/#using-nitrux-software-management) link to the [software management tutorial](https://nxos.org/tutorial/software-management/).
